src/Datadog.Trace.ClrProfiler.Native/lib/spdlog/include/spdlog/fmt/fmt.h

```diff
@@ -10,16 +10,18 @@
 // By default spdlog include its own copy.
 //
 
-#if 0 // !defined(SPDLOG_FMT_EXTERNAL)
```

Comment:
Was this change intentional? Don't have the context for why it was set like this previously